MyApplication/app/src/main/res/layout/activity_constraint.xml

```diff
@@ -0,0 +1,144 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/sub_ponit_guideline"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_begin="50dp" />
+
+    <ImageView
+        android:id="@+id/backgroud_image"
+        android:layout_width="match_parent"
+        android:layout_height="150dp"
```

Comment:
`layout_height="150dp"` will always be 20% of the screen? We want 20% of the screen irrespective of which device we are using. Please go through the `ConstraintLayout` concepts again and update the logic.